mediapicker/src/main/java/iam/thevoid/mediapicker/rxmediapicker/RxMediaPicker.java

```diff
@@ -199,13 +205,15 @@ public void dismiss() {
 
         private OnDismissListener onDismissListener;
 
-        private Duration videoMaxDuration = new Duration(15, TimeUnit.SECONDS);
```

Comment:
I don't like this. If user has not set this param he'll never pick the video

Reply:
He will. By my tests so far, I can shot any video for any size and any duration. If I don't change this, I always need to set a long duration for my movies (my MI A2 shots movies at 4k, and this limitation are hit so fast)